docs/source/user_guide/backends/omnisci.rst

```diff
@@ -80,14 +89,8 @@ Its main methods are:
 - `to_ibis`
 - `from_ibis`
 
-`parse` method ... # @TODO
-
-`to_ibis` method ... # @TODO
-
-`from_ibis` method ... # @TODO
-
-`MapDClient` class is used to connect to `omniscidb` database and manipulation data
-expression. Its main methods are:
+`MapDClient` class is used to connect to an `omniscidb` database and manipulate data
```

Comment:
```suggestion
`OmniSciDBClient` class is used to connect to an `omniscidb` database and manipulate data
```

Reply:
are these listed in the api section? e.g. so we can reference them